math.lua

```diff
@@ -75,14 +51,29 @@ function flib_math.round(num, divisor)
   end
 end
 
+--- Ceil a number to N decimal places.
```

Comment:
These functions don't round to N decimal places, they round to the nearest multiple of N.

Reply:
CopyPasta FTW